src/scss/cdb-icon-font.scss

```diff
@@ -798,436 +810,446 @@ All available icons in CartoDB.
+.CDB-IconFont-pencil::before {
   content: '\E04E';
 }
 
 
-.CDB-IconFont-mergeArrow::before {
+.CDB-IconFont-fountainPen::before {
   content: '\E04F';
 }
 
 
-.CDB-IconFont-mergeColumns::before {
+.CDB-IconFont-emptyDoc::before {
   content: '\E050';
 }
 
 
-.CDB-IconFont-mergeSpatial::before {
+.CDB-IconFont-mergeArrow::before {
   content: '\E051';
 }
 
 
-.CDB-IconFont-globe::before {
+.CDB-IconFont-mergeColumns::before {
   content: '\E052';
 }
 
 
-.CDB-IconFont-chevronDown::before {
+.CDB-IconFont-mergeSpatial::before {
   content: '\E053';
 }
 
 
-.CDB-IconFont-streets::before {
+.CDB-IconFont-globe::before {
   content: '\E054';
 }
 
 
-.CDB-IconFont-keys::before {
+.CDB-IconFont-chevronDown::before {
   content: '\E055';
 }
 
 
-.CDB-IconFont-facebookSquare::before {
+.CDB-IconFont-streets::before {
   content: '\E056';
 }
 
 
-.CDB-IconFont-fullscreen::before {
+.CDB-IconFont-keys::before {
   content: '\E057';
 }
 
 
-.CDB-IconFont-rArrowLight::before {
+.CDB-IconFont-facebookSquare::before {
   content: '\E058';
 }
 
 
-.CDB-IconFont-share::before {
+.CDB-IconFont-fullscreen::before {
   content: '\E059';
 }
 
 
-.CDB-IconFont-linkedin::before {
+.CDB-IconFont-rArrowLight::before {
   content: '\E05A';
 }
 
 
-.CDB-IconFont-calendarBlank::before {
+.CDB-IconFont-share::before {
   content: '\E05B';
 }
 
 
-.CDB-IconFont-creativeCommons::before {
+.CDB-IconFont-linkedin::before {
   content: '\E05C';
 }
 
 
-.CDB-IconFont-bubble::before {
+.CDB-IconFont-calendarBlank::before {
   content: '\E05D';
 }
 
 
-.CDB-IconFont-bars::before {
+.CDB-IconFont-creativeCommons::before {
   content: '\E05E';
 }
 
 
-.CDB-IconFont-arrowMenuLight::before {
+.CDB-IconFont-bubble::before {
   content: '\E05F';
 }
 
 
-.CDB-IconFont-tieFighter::before {
+.CDB-IconFont-bars::before {
   content: '\E060';
 }
 
 
-.CDB-IconFont-ray::before {
+.CDB-IconFont-arrowMenuLight::before {
   content: '\E061';
 }
 
 
-.CDB-IconFont-markup::before {
+.CDB-IconFont-tieFighter::before {
   content: '\E062';
 }
 
 
-.CDB-IconFont-table::before {
+.CDB-IconFont-ray::before {
   content: '\E063';
 }
 
 
-.CDB-IconFont-airlock::before {
+.CDB-IconFont-markup::before {
   content: '\E064';
 }
 
 
-.CDB-IconFont-pin::before {
+.CDB-IconFont-table::before {
   content: '\E065';
 }
 
 
-.CDB-IconFont-attache::before {
+.CDB-IconFont-airlock::before {
   content: '\E066';
 }
 
 
-.CDB-IconFont-download::before {
+.CDB-IconFont-pin::before {
   content: '\E067';
 }
 
 
-.CDB-IconFont-group::before {
+.CDB-IconFont-attache::before {
   content: '\E068';
 }
 
 
-.CDB-IconFont-dashboard::before {
+.CDB-IconFont-download::before {
   content: '\E069';
 }
 
 
-.CDB-IconFont-spinner::before {
+.CDB-IconFont-group::before {
   content: '\E06A';
 }
 
 
-.CDB-IconFont-mapsearch::before {
+.CDB-IconFont-dashboard::before {
   content: '\E06B';
 }
 
 
-.CDB-IconFont-rArrow::before {
+.CDB-IconFont-spinner::before {
   content: '\E06C';
 }
+
+
+.CDB-IconFont-mapsearch::before {
```

Comment:
@piensaenpixel points out that this is only the generated stylesheet, so we'll leave it like it's now for the moment :)